src/UI/Implementation/Component/Button/Close.php

```diff
@@ -5,8 +5,10 @@
 namespace ILIAS\UI\Implementation\Component\Button;
 
 use ILIAS\UI\Component as C;
+use ILIAS\UI\Implementation\Component\ComponentHelper;
 use ILIAS\UI\Implementation\Component\JavaScriptBindable;
 
 class Close implements C\Button\Close {
 	use JavaScriptBindable;
+	use ComponentHelper;
```

Comment:
No, I don't think so. It is just a default implementation, the "canonical name" could as well be different, think of the discussion about the hierarchy of factories we had today. I also deliberately did not introduce a base class for components, because this would force every implementation of a component to be in the same inheritance hierarchy, and I don't see benefit in that.